sound/soc/codecs/hdmi-codec.c

```diff
@@ -278,6 +278,7 @@ struct hdmi_codec_priv {
 	bool busy;
 	struct snd_soc_jack *jack;
 	unsigned int jack_status;
+	u8 iec_status[4];
```

Comment:
Maybe it would be better to have 5 byte array? Although fifth byte doesn't seem to be used usually, it at least nice to have this option in shared codec (DW-HDMI driver uses that fifth byte).

Reply:
Indeed, I changed it, thanks!